app/models/polling/dev_cert_status_updater.rb

```diff
@@ -1,5 +1,6 @@
 class DevCertStatusUpdater
   def update_hub_usage_status_for_cert(_hub_config_api, certificate)
-    CertificateInUseEvent.create(certificate: certificate)
+    time_cache_will_have_cleared_by = Time.now + Rails.configuration.hub_certs_cache_expiry
```

Comment:
It feels like if we are still having to work around the existence of a cache, then we haven't really got to the bottom of this issue. I'm not saying it's definitely worth worrying about: just that ideally, one would be able to make a request that definitively tells us which certs are actually being used (i.e. regardless of any possibility that the cache time is longer than we believe).

(Also, I'm obviously not saying that caching is wrong. Just that our updating should be able to interrogate the service to see which certs it is using - I'd have thought.)

Reply:
This is the accepted workaround as we have no easy way to interrogate all the 3 services (and their instances (x3)) without changing the hub infrastructure/load balancers or putting in some sort of pub/sub logic.